src/transformers/modeling_bert.py

```diff
@@ -436,17 +478,28 @@ def forward(
             attention_output = cross_attention_outputs[0]
             cross_attention_weights = cross_attention_outputs[1]
 
+        """
```

Comment:
are you skipping chunking for this version?

Reply:
For now. It wasn't compatible in a straightforward fashion so didn't want to invest too much time on it if the design is not approved by the team in the end.